src/templates/RecordTemplate.rs

```diff
@@ -0,0 +1,27 @@
+{#
```

Comment:
OOC, how come this (and inside `TopLevelFunctionTemplate`) is in a special `{#` comment block, but the others aren’t? Because we don’t want to include this in the generated output, it’s for the benefit of whoever’s editing the template?

Reply:
That's my interpretation FWIW; I think it makes sense considering that this comment was moved from outside of the top-level `for rec in ci.iter_record_definitions()` to inside it, so if we left it as a plain comment we'd get multiple copies of it, one per record.